style.css

```diff
@@ -337,3 +331,35 @@ header .links li:first-child {
   background-repeat: no-repeat;
   padding: .5em;
 }
+
+
+@media (max-width: 650px) {
+
+  header .links {
+    display: none;
+  }
+
+  .header, .footer {
+    background-color: #0c3757;
+    color: white;
+  }
+
+  main {
+    margin: 0;
+    flex-direction: column;
+  }
+
+  .sidebar {
+    width: 100%;
+    margin: 0;
```

Comment:
Finally I add the flexbox rule. It's a lot more better. And I modify the borders